src/js/burials/containers/ConfirmationPage.jsx

```diff
@@ -44,7 +44,7 @@ class ConfirmationPage extends React.Component {
     const { deathCertificate, transportationReceipts } = form.data;
 
     return (
-      <div className="edu-benefits-submit-success">
+      <div className="benefit-submit-success">
```

Comment:
Do we even need this class? It seems like we could make a common `page-title` style that does the centering and have the tests look for that.